src/reducers/contactsReducer.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
+import { FETCH_CONTACTS } from '../actions/types';
+
+export default (state = {}, action) => {
```

Comment:
Here the default for `state` should not be an empty object. You can declare above this an `initialState` constant and set some default values there. For example:
```
const initialState = {
  contacts: []
}

export default (state = initialState, action) => {
....................................
}
```
This helps getting rid of errors like if you have no contacts in the list and you try to do some array operations on that contacts array. If it's null you'll get errors. If it's an empty array, it's fine. Default values can be whatever you want to have as default for various state properties.